modules/sso-rbac/locals.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_ssoadmin_instances" "this" {}
+
+locals {
```

Comment:
why we need multiple locales{} blocks here,
you can combine all locals in this file just one locales{} block